src/common/eltwise.cpp

```diff
@@ -42,7 +42,7 @@ status_t eltwise_desc_init(eltwise_desc_t *eltwise_desc, prop_kind_t prop_kind,
                   eltwise_exp, eltwise_gelu, eltwise_swish)
         && IMPLICATION(prop_kind == backward_data, diff_data_desc != nullptr)
         && IMPLICATION(one_of(data_desc->data_type, mkldnn_s32, mkldnn_s8,
-                    mkldnn_u8), alg_kind == eltwise_relu && alpha == 0);
+                    mkldnn_u8), alg_kind == eltwise_relu );
```

Comment:
why was this restriction removed?

Reply:
Sorry, I missed that int relu does not support negative slopes. 
Tests need to be fixed. 
I need an advise - should I remove jit code that supports non-zero alpha?

Reply:
I don't know the reason behind why int8 relu only support zero negative slope. Trying to find. 

But for the sake of this MR. I suggest keeping the restriction of `alpha=0` and we can keep the support for non-zero alpha in jit code too, since in reference eltwise also we have support for non-zero alpha. 

If this is fine with you, I can revert this change (i.e. go back to `alpha==0`) and  resolve conflicts with latest master and then run CI and proceed with merging this MR.

Reply:
I've resolved with the latest master. 
Reverted alpha==0 restriction in eltwise.cpp
And force alpha==0 for int relu in test. 
Please, take a look